examples/rfmip-clear-sky/Makefile

```diff
@@ -5,8 +5,8 @@ RRTMGP_BUILD = $(RRTMGP_ROOT)/build
 #
 # RRTMGP library, module files
 #
-LDFLAGS   += -L$(RRTMGP_BUILD)
-LIBS      += -lrrtmgp -lrte
+# LDFLAGS   += -L$(RRTMGP_BUILD)
```

Comment:
Why are these removed?

Reply:
The Makefile is linking these static libraries twice: librrtmgp and librte.a

Firstly here:
```
rrtmgp_rfmip_sw:   rrtmgp_rfmip_sw.o   $(ADDITIONS) $(RRTMGP_BUILD)/librrtmgp.a $(RRTMGP_BUILD)/librte.a`
```

then secondly here:
```
LDFLAGS   += -L$(RRTMGP_BUILD)
LIBS      += -lrrtmgp -lrte
```

Again, this is a feature/weakness of the Cray linker and other linkers that are partly-based on LLVM and support OpenMP for GPUs. They are just not as robust as their CPU brethren. Because of the duplicate linkage, the binary cannot be built. 

So, the take home message is that the GPU linker is less robust than the CPU linker. The static libraries can only be linked once and must be in a specific order.